index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Busiwas Portfolio</title>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="style.css" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav-bar">
+            <a href=""><i class="fa fa-bars hamburger" aria-hidden="true"></i></a>
+        </nav>
+    </header>
+<script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script> 
+<main>
+    <section class="section-1">
+        <h1 class="heading-1">
+            Busiwa Liuma
+        </h1>
+        <p class="par-1">
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+            <button class="button-1" type="button">Start Collaboration</button>
+        <ul class="sec-icons">
+            <li><a href=""><i class="fab fa-angellist"></i></a></li>
+            <li><a href=""><i class="fab fa-facebook"></i></a></li>
+            <li><a href=""><i class="fab fa-github"></i></a></li>
+            <li><a href=""><i class="fab fa-twitter"></i></a></li>
+            <li><a href=""><i class="fab fa-instagram"></i></a></li>
+        </ul>   
```

Comment:
- [ ] These icons are end of the page in design not here. You can delete these.
- [ ] You should add background image instead of icons.

**Original design.**
![image](https://user-images.githubusercontent.com/38635158/125690836-2d35a38e-07df-447c-9a38-decbcc383968.png)
___

**Your design:**
![image](https://user-images.githubusercontent.com/38635158/125690904-4e5fdf24-5dc1-4daa-b5cc-c1872acde96a.png)